features/step_definitions/shared_steps.rb

```diff
@@ -11,23 +11,44 @@
   end
 end
 
-When(/^I click "(.+)" button$/) do |button|
-  click_button(button)
+# When(/^I click "(.+)" button$/) do |button|
+#   click_button(button)
+# end
+
+Then('I click {string} button') do |button|
+  sci_click_on_button(text: button)
+end
+
+Then('I click {string} icon') do |icon_class|
+  sci_click_on_icon(icon_class: icon_class)
 end
 
-Then('I click {string} Scinote button') do |button|
-  find('.btn', text: button).click
+Then('I click {string} button at position {int}') do |button, position|
+  sci_click_on_button(text: button, position: position)
 end
 
-Then('I click on {string} button') do |button|
-  find('.btn', text: button).click
+Then('I click {string} icon at position {int}') do |icon_class, position|
+  sci_click_on_icon(icon_class: icon_class, position: position)
 end
 
-Given('I click on {string} class button') do |button6|
-  find('.btn', class: button6, match: :first).click
+#
+# Then('I click {string} icon with delay') do |icon_class|
+#   sci_click_on_icon(icon_class: icon_class, wait: 1)
+# end
+
+# Then('I click on {string} button') do |button|
+#   find('.btn', text: button).click
+# end
+
+# Given('I click on {string} class button') do |button6|
+#   find('.btn', class: button6, match: :first).click
+# end
+
+Then('I click element with css {string}') do |selector|
+  find(selector).click
 end
 
-Given('I click on {string} id button') do |button1|
```

Comment:
Where we need it?

Reply:
What you mean exectly? `Given('I click on {string} id button') do |button1|` was removed?